howto/install/deploy-bare-metal.md

```diff
@@ -0,0 +1,93 @@
+To deploy Anbox Cloud on a public cloud (such as AWS, Azure or Google) or using MAAS or OpenStack, see the instructions in [Deploy Anbox Cloud with Juju](https://discourse.ubuntu.com/t/deploy-anbox-cloud-with-juju/17744).
+
+Alternatively, you can follow the instructions in this document to use the [manual cloud provider](https://jaas.ai/docs/manual-cloud) that Juju offers. This method allows you to deploy Anbox Cloud with Juju on a set of SSH connected machines.
+
+## Prerequisites
+
+Before you start the installation, ensure that you have the required prerequisites:
+
+* At least three Ubuntu 18.04 LTS or 20.04 LTS machines. See [Minimum hardware](https://discourse.ubuntu.com/t/requirements/17734#minimum-hardware) for details and recommendations.
+* Your *Ubuntu Advantage for **Applications*** token. If you don't have one yet, [speak to your Canonical representative](https://anbox-cloud.io/contact-us). If you already have a UA Applications token, sign in on https://ubuntu.com/advantage to retrieve it.
+  [note type="caution" status="Warning"]The *Ubuntu Advantage for **Infrastructure*** token that every user gets for free for personal use does **NOT** work and will result in a failed deployment. You must purchase a *Ubuntu Advantage for **Applications*** subscription by [contacting Canonical](https://anbox-cloud.io/contact-us).[/note]
+
+## Install Juju
+
+Juju is a tool for deploying, configuring and operating complex software on public or private clouds.
+
+You must install a Juju client on the machine that you use to run the deployment commands. To install Juju 2.9, enter the following command:
+
+    sudo snap install --channel=2.9/stable juju
+
+See [Juju version](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750#juju-version) for information about which Juju version is required for your version of Anbox Cloud.
```

Comment:
Is it correct that we link to the upgrade page?

Reply:
It's currently correct, yes. It doesn't really make sense though - I created a task to fix it.